gnomad_qc/v3/sample_qc/subpop_analysis.py

```diff
@@ -195,7 +195,6 @@ def main(args):  # noqa: D103
                 if args.test
                 else filtered_subpop_qc_mt(pop),
                 overwrite=args.overwrite,
-                _read_if_exists=not args.overwrite,
```

Comment:
Confirming you want this change? seems like the plan was to read if this checkpoint exists unless overwrite is supplied. And this may be a difficult computation that we want to use a checkpoint if available, but I may be missing something

Reply:
it's a write rather than a checkpoint so the _read_if_exists wouldn't work, the next step uses hl.read_matrix_table

Reply:
Ah yes, makes sense, this was split from a checkpoint to a write and read